src/algorithms/SortingAlgorithms.java

```diff
@@ -18,20 +26,59 @@ public static void sortByBubbleSorting(int[] array) {
         }
     }
 
-    public static void sortBySelectionSorting(int[] array){
-
+    public static void sortBySelectionSorting(int[] array) {
```

Comment:
It is not a selection sort.
Example
public static void selectionSort(int[] arr){
        for (int i = 0; i < arr.length - 1; i++) {
            int index = i;
            for (int j = i + 1; j < arr.length; j++){
                if (arr[j] < arr[index]){
                    index = j;
                }
            }
            int smallerNumber = arr[index];
            arr[index] = arr[i];
            arr[i] = smallerNumber;
        }
    }